exporter/exporterwrapper/exporterwrapper.go

```diff
@@ -34,6 +34,12 @@ import (
 	spandatatranslator "github.com/census-instrumentation/opencensus-service/translator/trace/spandata"
 )
 
+// OCSpanExporter is an interface for the ExportSpan function of trace.Exporter.
```

Comment:
Well, I'm open to a better approach, but this was what I seemed to need to do to be able to pass in a fake span exporter for a test. What would you suggest?

Reply:
I expected that you could do the same with https://github.com/census-instrumentation/opencensus-go/blob/b4a14686f0a98096416fe1b4cb848e384fb2b22b/trace/export.go#L30-L33 since it is exact the same interface... perhaps I'm missing something...

Reply:
Well, as far as I can tell, the `opencensus-go` package is not currently a build dependency of `opencensus-service`. Would it be worth adding it for importing that interface?

Reply:
@pjanotti friendly ping on this. Are you OK with the new interface to avoid the `opencensus-go` import? If so, could you merge this PR?